### ISSUE TITLE
Hide keyboard in delegate - (void)scrollViewWillBeginDragging:(UIScrollView *)scrollView

### ISSUE THREAD

**Issue by fibers** [jessesquires/JSQMessagesViewController#993]
I use the following the code to hide the keyboard when the user start to scrolling the collection view.

```
- (void)scrollViewWillBeginDragging:(UIScrollView *)scrollView{
if([self.inputToolbar.contentView.textView isFirstResponder]){
        [self.inputToolbar.contentView.textView resignFirstResponder];
    }
}
```

But the inputToolbar will be moved to the bottom first, then the keyboard hide after a half second.

This is weird, any comments will be appreciated.

**Comment by cnowak7** [comment]
You want the the keyboard dismissed once you start scrolling to another view?

**Comment by wdcurry** [comment]
@cnowak7: i am assuming he means that when a user starts scrolling the collection view itself, thus viewing more messages on the active view.

**Comment by cnowak7** [comment]
Oh ok, well either way, when I do `[self.view endEditing:YES]`, this dismisses the keyboard for me. I hope this helps :)

**Comment by jessesquires** [comment]
Fixed by 752fd38aa8b899cbe2e21b6595de58920977bfc2

**Comment by jessesquires** [comment]
You can now use `.keyboardDismissMode`
